zzzz_modular_occulus/code/modules/projectiles/guns/projectile/rifle/whitenail.dm

```diff
@@ -42,16 +43,17 @@
 		generate_ammo()
 
 /obj/item/weapon/gun/projectile/automatic/whitenail/proc/generate_ammo() // generates ammo in the loaded mag
```

Comment:
Similarly, this proc can be reworked, generating up to the max ammo count and no higher using `min(ammo_magazine.max_ammo - src.get_ammo(), 3)`.